blog-sync.mjs

```diff
@@ -1,3 +1,4 @@
+// Can't understand how to convert this file to ts 🤷‍
```

Comment:
Yeah, agree. 

@kamilkisiela we talked about the same thing yesterday, right? writing a TS file and use ESM

Reply:
I tried to make it work but failed :)

Reply:
I think it's fine to run `blog-sync.mjs` with `node` and turn `authors.js` to `authors.ts` + add `ts-node/register` (with `"module": "esnext"`).